api/cloudcontroller/ccv3/buildpack.go

```diff
@@ -173,98 +60,29 @@ func (client Client) UpdateBuildpack(buildpack Buildpack) (Buildpack, Warnings,
 		Body:        bytes.NewReader(bodyBytes),
 		URIParams:   map[string]string{"buildpack_guid": buildpack.GUID},
 	})
-	if err != nil {
-		return Buildpack{}, nil, err
-	}
-
-	var responseBuildpack Buildpack
-	response := cloudcontroller.Response{
-		DecodeJSONResponseInto: &responseBuildpack,
-	}
-	err = client.connection.Make(request, &response)
 
-	return responseBuildpack, response.Warnings, err
+	return responseBody, warnings, err
```

Comment:
actually wouldn't responseBody be undefined here?

Reply:
My bad, thanks for noticing it.
It should be ok now